shared/featureconfig/config.go

```diff
@@ -0,0 +1,65 @@
+/*
+Package featureconfig defines which features are enabled for runtime
+in order to selctively enable certain features to maintain a stable runtime.
+
+The process for implementing new features using this package is as follows:
+	1. Add a new CMD flag in flags.go, and place it in the proper list(s) var for its client.
+	2. Add a condition for the flag in the proper Configure function(s) below.
+	3. Place any "new" behavior in the `if flagEnabled` statement.
+	4. Place any "previous" behavior in the `else` statement.
+	5. Ensure any tests using the new feature fail if the flag isn't enabled.
+	5a. Use the following to enable your flag for tests:
+			cfg := &FeatureFlagConfig{
+				VerifyAttestationSigs: true,
+			}
+			InitFeatureConfig(cfg)
```

Comment:
This formatting is off

![image](https://user-images.githubusercontent.com/7246818/54497195-15570c80-48ce-11e9-8f8a-a36e794d24d2.png)